node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -72,6 +73,7 @@ const (
 	WaitForMpathWaitIntervalSec = 1
 	FcHostSysfsPath             = "/sys/class/fc_remote_ports/rport-*/port_name"
 	IscsiHostRexExPath          = "/sys/class/iscsi_host/host*/device/session*/iscsi_session/session*/targetname"
+	sysDeviceSymLinkToLun       = "/sys/block/%s/device"
 	deviceDeletePathFormat      = "/sys/block/%s/device/delete"
```

Comment:
consider using `sysDeviceSymLinkToLun` in `deviceDeletePathFormat` definition

Reply:
done